meta-dts-distro/recipes-tests/converged-security-suite/txt-suite_2.6.0.bb

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "Converged Security Suite's txt-suite"
+HOMEPAGE = "https://github.com/9elements/converged-security-suite"
+SECTION = "devel"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
+
+SRC_URI = "https://github.com/9elements/converged-security-suite/releases/download/v2.6.0/txt-suite"
```

Comment:
Would be great to have bg-suite as well

Reply:
Issue: https://github.com/Dasharo/meta-dts/issues/39